releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,24 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 7.0.0 - November 21, 2023
+### Breaking Changes
+* Removed .NET Core 3.1 as a target framework as .NET Core 3.1 is out of support. Updated `Microsoft.Extensions` dependencies to version 6.0.0 and later. [#482](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/482)
```

Comment:
Dropping netcoreapp3.1 is a breaking change for `Microsoft.Azure.AppConfiguration.AspNetCore`, not for this package.

What are those `Microsoft.Extensions.*` changes really for?

Reply:
I guess the package versions don't really need to be mentioned. I can move this to the AspNetCore file and mention the inclusion of .netstandard2.1 as a target here instead?